tests/integration/test_issue_2235.py

```diff
@@ -0,0 +1,67 @@
+import os
+import subprocess
+import sys
+
+import pytest
+
+from pex.compatibility import PY3
+from testing import PY38, ensure_python_interpreter
+
+
+@pytest.mark.skipif(sys.version_info < (3, 8), reason="Flask 2.3.3 requires Python3.8+.")
+def test_standard_library_is_included(
+    tmpdir,  # type: Any
+    pex_project_dir,  # type: str
+):
+    # type: (...) -> None
+
+    # N.B.: The package script requires Python 3.
+    python = sys.executable if PY3 else ensure_python_interpreter(PY38)
```

Comment:
You already guaranteed Python>=3.8 via the skipif above; so this conditional is not needed.